Exercise_D.js

```diff
@@ -14,7 +14,17 @@
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
-}
+  if(number < 0 || number == 0 || isNaN(number) == true) {
+    return "Error";
+  }else if(number >= 1000000) {
+    return number;
+  }else {
+    while(number<1000000) {
+      number = number*10;
+      console.log(number)
+    }
+    return number;
+    }
+  }
 
-console.log(multiplyNumber(10));
+console.log(multiplyNumber(900));

```

Comment:
For this exercise, when run the output is supposed to give only one output instead it gives out 6 outputs as you have inside a loop where it multiplies the number until it reaches a value bigger than 1000000. To correct this, first, we can use the part where we have to check if the number is smaller than and equal to 0 by using the <= sign and then using || isNaN to check if it's not a number.